lib/rules/selector-max-specificity/__tests__/index.mjs

```diff
@@ -65,6 +65,9 @@ testRule({
 		{
 			code: ':nth-child(2n + 1) {}',
 		},
+		{
+			code: '[value] {}',
```

Comment:
nit

It's kind of contradictory.
i.e. that's the attribute name not its value